packages/api/src/checkTypes.manual.ts

```diff
@@ -28,17 +28,18 @@ async function derive (api: ApiPromise): Promise<void> {
     console.log('current author:', header.author);
   });
 
-  await api.query.staking.intentions((intentions): void => {
-    console.log('intentions:', intentions);
+  await api.query.staking.validatorCount((count): void => {
```

Comment:
removed `intentions` because it was underlined red

Reply:
This actually is a v1 thing - so it should not be underlined red, it can exist. It is a valid test actually, the ones that it doesn't know about, it shouldn't complain about.

We should also add some modules-not-defined, i.e. `api.query.parachains` - similar to the above, don't complain about modules not defined.

TL;DR -

- don't complain about methods not known, but allow the `<Type>` overrides
- don't complain about modules not known, also allow the generic overrides